### ISSUE TITLE
Bulk Update Multiple Tables and Columns via Single CSV File

### ISSUE THREAD

**Issue by noelaum** [open-metadata/OpenMetadata#16729]
We are currently using OMD v1.4.2 and testing a bulk update on Data Assets (description, Owner, Tier, Domain, etc.).

Since the OMD API does not support bulk importing of multiple tables with a single CSV file, we would like to know if it is possible to add this feature to the OMD API. Appreciate your feedback on this. Thanks!

**Comment by harshach** [comment]
you can do multiple tables against a database schema

**Comment by noelaum** [comment]
Thanks for your input, @harshach. Do you have a step-by-step guide on how to do it? I'm new to working with APIs?
Is this documentation can help? https://docs.open-metadata.org/swagger.html#operation/importDatabaseSchema

**Comment by harshach** [comment]
@noelaum this is question in support channel and not for github issues. Please open a github issue once confirmed its a bug or a feature request

**Comment by noelaum** [comment]
@harshach ,I already raised this concern, and according to the support team, the OMD API does not support bulk importing of multiple tables with a single CSV file. Here's the link to the Slack conversation: https://openmetadata.slack.com/archives/C02B6955S4S/p1718864234398369?thread_ts=1718680131.869139&cid=C02B6955S4S

I suggested adding this feature to the OMD API, and they advised me to file a feature request. Thanks!
